src/inputs/DateFields/DateField.mock.tsx

```diff
@@ -1,6 +1,6 @@
 import { format, parse } from "date-fns";
 import { useState } from "react";
-import { DateFieldProps } from "src/inputs";
+import { DateFieldProps } from "src/inputs/index";
```

Comment:
I kinda like leaving off the `/index`? Although iirc file extensions are becoming required soon anyway, so will `src/inputs` just stop working at that point? :thinking: Not sure.

Reply:
Probably was an accident when updating folder structure. Can update.